configure.ac

```diff
@@ -5152,8 +5152,12 @@ AC_SEARCH_LIBS(dlopen, dl)
 #
 export_link_flags=--export-dynamic
 case $host_os in
-	mingw*|cygwin*)
+mingw*|cygwin*)
 	export_link_flags=--export-all-symbols
+	;;
+*-*-darwin*)
+	export_link_flags=--export_dynamic
```

Comment:
I think belongs elsewhere, will tell you later.

Reply:
Clarification, sorry: for cosmetics. It is ok here.
What I had in mind though is editing this line right below instead:
```
AX_APPEND_LINK_FLAGS([-Wl,$export_link_flags -Wl,-Bsymbolic -bsymbolic -Wl,-z,now -Wl,-bind_at_load])
```

Reply:
(and I cannot remove labels)

Reply:
And I guess I see why you did this way too, ok.

Reply:
The `AX_APPEND_LINK_FLAGS` line should probably say it is space delimited.
It is confusing to read, with the commas.

Reply:
I think it's okay as-is. I agree it'd be nice to comment on the line for clarity but I don't think it's worth another CI run; if I have to update this for something else I'll add that.